taichi/ir/statements.h

```diff
@@ -450,13 +521,17 @@ class LocalStoreStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE;
 };
 
+/**
+ * Same as "if (cond) true_statements; else false_statements;" in C++.
+ * "true_mask" and "false_mask" are used to support vectorization.
```

Comment:
Let's unify the style for quoting a variable/member.
```suggestion
 * |true_mask| and |false_mask| are used to support vectorization.
```